charts/README.md

```diff
@@ -182,6 +254,43 @@ The following table lists the configurable parameters of the latest Azure Disk C
 | `windows.resources.azuredisk.requests.memory`            | azuredisk memory requests limits                | 20Mi                                                           |
 | `cloud`                                           | cloud environment driver is running on                     | `AzurePublicCloud`                                                  |
 
-## troubleshooting
- - Add `--wait -v=5 --debug` in `helm install` command to get detailed error
- - Use `kubectl describe` to get more info
+### New or Updated Parameters for V2
```

Comment:
As I said, I'd like to keep the V2 driver info on the landing page. Since the driver parameters for V1 and V2 are quite large, perhaps they both should be moved to a separate page?